src/client/js/components/ModelHandler.js

```diff
@@ -133,6 +141,14 @@ export function setMonthlyAdjustment(month, value) {
     store.set(month + storeConfig.keys.monthlyAdjustment, value);
 }
 
+export function getTaskTypes() {

```

Comment:
Task types should be collected from last few days' logs instead of storing them separately. This might require UI changes first.